src/app/Plans/PlansPage.tsx

```diff
@@ -28,9 +28,11 @@ const plans = MOCK_PLANS;
 const migrations = MOCK_MIGRATIONS;
 
 const PlansPage: React.FunctionComponent = () => {
+  const history = useHistory();
   const providersQuery = useProvidersQuery();
 
-  const vmwareProviders = providersQuery.data?.vsphere || [];
+  // const vmwareProviders = providersQuery.data?.vsphere || [];
```

Comment:
Can you revert this change and go back to using the real data for `vmwareProviders`?

Reply:
Darn me!